docs/recipes/ReadUintFromContract.md

```diff
@@ -0,0 +1,197 @@
+---
+sidebar_position: 3
+title: Read a uint from a contract
+description: Learn how to read from contract functions which accepts arguments / no arguments and display them on UI.
+---
+
+This recipe demonstrates how to read data from contract functions that accept some arguments (parameters) or no arguments at all and displays it on the UI.
```

Comment:
Sometimes when I read this sentence it looks strange, maybe because it's too large? (Other times I read it and seems OK heh).

What about: "This recipe demonstrates how to read data from contract functions and display it on the UI. We'll showcase and example that accept some arguments (parameters), and another with or no arguments at all"

@edakturk14 could you also take a look with your writer's eyes? 🙃

Reply:
> It's much more clear + readable :D

💯💯,  Tysm Pablo 🙌